src/shared/modules/discovery/discoveryDuck.js

```diff
@@ -156,7 +168,11 @@ export const discoveryOnStartupEpic = (some$, store) => {
               getAllowedBoltSchemes(store.getState()),
               host
             )
-            store.dispatch(updateDiscoveryConnection({ host })) // Update discovery host in redux
+
+            const isAura = getEnv(store.getState()) === CLOUD
+            const supportsMultiDb =
+              !isAura && parseInt((result.neo4j_version || '0').charAt(0)) >= 4
```

Comment:
Aren't there Aura editions that support multi-db?

Reply:
From asking there's only editions with proper support for multi-db. At most they support the system-db in addition to the default one. So I think if it's important there, you'll have to use the url param